### ISSUE TITLE
Touchpoints modal on static site doesn't maintain focus

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4030]
## Area or Component
Static site - any page

## Quick Summary
The modal for Touchpoints doesn't hold focus, so screenreaders can read outside it and users can tab outside the modal.

### How to Replicate
1. Open Touchpoints on the static site
2. Start tabbing backwards
3. See that you go back to the main page content

## Accessibility Violation
Able to browse outside modal with screen reader: https://cdc-devtools.dequecloud.com/rule-help/en/reading-order-browse-outside-modal
Keyboard focus is not maintained in modal: https://cdc-devtools.dequecloud.com/rule-help/en/focus-modal-moves-outside

## Action Requested
Ensure that keyboard focus can't escape the Touchpoints modal.

## Screenshots (optional)
![Home _ SimpleReport](https://user-images.githubusercontent.com/80282552/180336572-8dc26902-63a4-46fa-8649-4e7b2648a6d8.gif)

## Original Deque Issue
https://docs.google.com/spreadsheets/d/16udNpwdJ8sltAULnmSIGgkWVQsfuFvcoymP0-s_QwfE/edit#gid=856291185&range=7:8
